### ISSUE TITLE
Clarification on behavior when user is removed from Microsoft Entra group associated to a rule.

### ISSUE THREAD

**Issue by JPGRodrigues** [MicrosoftDocs/azure-devops-docs#13969]
On the documentation we have "When users leave the Microsoft Entra group, Azure DevOps automatically removes them."
Shouldn't this be restated as "When users leave the Microsoft Entra group, Azure DevOps automatically removes the license referenced by the group rule." ? Some customers are reading this as the user is removed from Azure DevOps!

Can we also get clarification what final access conditions will be granted to the user after the rule evaluation?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: ab08f86b-6991-b3f1-b07e-51048c503bc8
* Version Independent ID: 31556922-2ea2-791e-7508-6590762a0365
* Content: [Add group rule to assign access levels - Azure DevOps Services](https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/assign-access-levels-by-group-membership?view=azure-devops)
* Content Source: [docs/organizations/accounts/assign-access-levels-by-group-membership.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/organizations/accounts/assign-access-levels-by-group-membership.md)
* Service: **azure-devops**
* Sub-service: **azure-devops-organizations**
* GitHub Login: @chcomley
* Microsoft Alias: **chcomley**

**Comment by WilliamAntonRohm** [comment]
@JPGRodrigues -- João, thank you for your feedback and questions.  Please consider these resources:
- [Azure DevOps Services](https://azure.microsoft.com/support/devops/) 
- [Azure DevOps on Stack Overflow](https://stackoverflow.com/questions/tagged/azure-devops)

@chcomley -- Chrystal, please look into this issue.

**Comment by chcomley** [comment]
Hi @JPGRodrigues, thanks for reaching out. Sorry for the delay. I've updated the article, which should be live shortly. Here's the information:
**User Removal from Azure DevOps:** The statement in the documentation that reads, “When users leave the Microsoft Entra group, Azure DevOps automatically removes them,” can indeed be misinterpreted. To clarify, when a user leaves the Microsoft Entra group, Azure DevOps does not remove the user entirely from the platform. Instead, it adjusts their access level based on the rules defined for that group. The user remains in Azure DevOps but may have different permissions or access rights.
**Final Access Conditions After Rule Evaluation:** After the rule evaluation, the user’s access conditions are determined by the highest access level assigned to them. If a user is part of multiple rules or Microsoft Entra groups with varying access levels (e.g., Stakeholder and Basic), they will have the Basic access level. In other words, the most permissive access level prevails. 
To summarize:

- Leaving a group doesn’t remove the user entirely.
- The user’s access level is based on the rules and groups they belong to.
- The highest access level assigned to the user determines their final permissions. 

Thanks so much!
